system-test/bigtable.ts

```diff
@@ -1417,6 +1417,66 @@ describe('Bigtable', () => {
 
       Object.keys(policy).forEach(key => assert(key in updatedPolicy));
     });
+    describe('copying backups', () => {
+      async function testCopyBackup(backup: Backup, newBackup: Backup) {
```

Comment:
I would like to see some more assertions here. I am thinking something along the lines of:
1. copy the backup, wait for operation to complete
2. list backups
3. assert the copied backup is in the list, and assert it has correct name, expire time, etc

Reply:
list backups contains name and id so I made an assert that compares those two and ensures exactly one new backup gets created that matches.